src/sidebar/components/thread-card.js

```diff
@@ -33,10 +33,24 @@ function ThreadCard({ frameSync, settings = {}, thread }) {
     [frameSync]
   );
 
+  /**
+   * Is the target's event an <a> or <button> element, or does it have
+   * either as an ancestor?
+   */
+  const isFromButtonOrLink = target => {
+    return !!target.closest('button') || !!target.closest('a');
```

Comment:
We'll need to use the [`closest` polyfill](https://github.com/hypothesis/client/blob/0d15ec5dfcdec56b6394ebf1e1daa8bf9572edbf/src/annotator/util/dom-element.js#L11) for IE 11 support since we won't be dropping support for that browser until July 1st. You'll need to move that file to `src/shared/` since files in `src/sidebar` are not supposed to depend on modules in `src/annotator` and vice-versa.

Reply:
okay, I moved this to /shared for the time being so it follows our standard between sidebar and annotator